src/etc/di.xml

```diff
@@ -14,4 +14,6 @@
                 type="ScandiPWA\WishlistGraphQl\Model\Resolver\WishlistResolver"/>
     <preference for="Magento\WishlistGraphQl\Model\Resolver\WishlistItemsResolver"
                 type="ScandiPWA\WishlistGraphQl\Model\Resolver\WishlistItemsResolver"/>
+    <preference for="Magento\WishlistGraphQl\Model\Resolver\WishlistItems"
+                type="ScandiPWA\WishlistGraphQl\Model\Resolver\WishlistItemsResolver"/>
```

Comment:
How looks the original file? Do we really need to override it fully without extending it?
At all, what for this preference?